src/tests/Rockets.test.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from '../redux/configureStore';
+import Rockets from '../components/Rockets/Rockets';
+import { fetchRockets, rocketBooking, rocketCancelation } from '../redux/rockets/rockets';
+
+describe('Rocket renders correctly', () => {
+  test('Renders rockets Correctly', () => {
+    const rocket = render(
+      <Provider store={store}>
+        <Router>
+          <Rockets />
+        </Router>
+      </Provider>,
+    );
+    expect(rocket).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Please try to either remove or add passing tests, please

![image](https://user-images.githubusercontent.com/68030297/202587269-38aca9ea-6b1a-4614-8845-6e38e5ba8cc2.png)
---

for more information about "obsolete" please follow this link about [What are obsolete snapshots and snapshot files?](https://stackoverflow.com/questions/57793527/what-are-obsolete-snapshots-and-snapshot-files)

Reply:
Okay, thank you @adel-gu